sirepo/feature_config.py

```diff
@@ -134,15 +132,21 @@ def b(msg, dev=False):
         f'{i}: cannot be in proprietary_sim_types and default_proprietary_sim_types'
     s = set(
         _cfg.sim_types or (
-            PROD_FOSS_CODES if pkconfig.channel_in('prod') else _FOSS_CODES
+            PROD_FOSS_CODES if pkconfig.channel_in('prod') else FOSS_CODES
         )
     )
     s.update(_cfg.proprietary_sim_types, _cfg.default_proprietary_sim_types)
     for v in _DEPENDENT_CODES:
         if v[0] in s:
             s.add(v[1])
-    x = s.difference(VALID_CODES)
-    assert not x, \
-        'sim_type(s) invalid={} expected={}'.format(x, VALID_CODES)
     _cfg.sim_types = frozenset(s)
+    if 'sirepo' not in _cfg.package_path:
+        _cfg.package_path = (*_cfg.package_path, 'sirepo')
```

Comment:
[Previous comment](https://github.com/radiasoft/sirepo/pull/3747/files#r687068441) was not this. We don't always want to inject sirepo. Rather, if there is a package_path, it's explicitly listed. So remove this code, and add this above:

```py
package_path=(
    ('sirepo',),
```